core/src/main/java/com/flipkart/gjex/core/filter/Filter.java

```diff
@@ -15,64 +15,36 @@
  */
 package com.flipkart.gjex.core.filter;
 
-import com.google.protobuf.GeneratedMessageV3;
-import io.grpc.Metadata;
-import io.grpc.Status;
-import io.grpc.StatusRuntimeException;
+import com.flipkart.gjex.core.logging.Logging;
 
 /**
- * A Filter interface for processing Request, Request-Headers, Response and Response-Headers around gRPC method invocation
- *  
- * @author regu.b
+ * A Filter interface for processing Request, Request-Headers, Response and Response-Headers 
+ * around gRPC and HTTP method invocation
  *
- * @param <Req> Proto V3 message
- * @param <Res> Proto V3 message
+ * @author ajay.jalgaonkar
```

Comment:
Please dont change the original author information. Since there has been no new code here.

Reply:
reset